File: src/badge-gen.py
#include <stdio.h>
import os
directory = './badges'
for filename in os.listdir(directory):
    f = os.path.join(directory, filename)
    if os.path.isfile(f):
        print("<img alt=\""+f+"\"  title=\""+f+"\" src=\""+f+"\">",end =" ")

File: src/badge-gen.lua
local path = "./badges/"
local badges = io.popen("dir "..path)
if badges then
  for i in badges:read("*a"):gmatch("(.-)\n") do 
    for w in i:gmatch("%S+") do 
      io.write("<img alt=\""..path..w.."\" src=\""..path..w.."\">"); 
    end 
  end
end
